pkg/signal/signal_unix.go

```diff
@@ -8,5 +8,9 @@ import (
 
 // Signals used in api/client (no windows equivalent, use
 // invalid signals so they don't get handled)
+
+// signal sent to a process when a child process terminates, is interrupted, or resumes after being interrupted.
 const SIGCHLD = syscall.SIGCHLD
+
+// signal sent to a process when its controlling terminal changes its size
 const SIGWINCH = syscall.SIGWINCH
```

Comment:
I thought so but ``golint`` is quite happy about it. It might have something to do with the build flags (my naive guess).